build_library/release_util.sh

```diff
@@ -42,6 +40,38 @@ DEFINE_string sign "" \
   "Sign all files to be uploaded with the given GPG key."
 DEFINE_string sign_digests "" \
   "Sign image DIGESTS files with the given GPG key."
+DEFINE_string image_compression_format "${DEFAULT_IMAGE_COMPRESSION_FORMAT}" \
+  "Compress the resulting images using this format. Options are: none, bz2, gz, zip"
+
+
+compress_file() {
+    local filepath="$1"
+
+    [ ! -f "${filepath}" ] && die "Image file ${filepath} does not exist"
+
+    case "${FLAGS_image_compression_format}" in
+    "none"|"")
+        echo -n "${filepath}"
+        return 0
+        ;;
+    "bz2")
+        IMAGE_ZIPPER="lbzip2 --compress --keep"
+        ;;
+    "gz")
+        IMAGE_ZIPPER="pigz --keep"
+        ;;
+    "zip")
+        IMAGE_ZIPPER="pigz --keep --zip"
+        ;;
+    *)
```

Comment:
could you also add support for zstd?

Reply:
Had the same thought but we can also add it later and not as part of this PR

Reply:
make sense

Reply:
Seems like the binary is already in the sdk container, so why not? :D